src/current/_includes/v23.1/sidebar-data/faqs.json

```diff
@@ -0,0 +1,35 @@
+{
+    "title": "FAQs",
+    "is_top_level": true,
+    "items": [{
+        "title": "FAQs",
```

Comment:
Can we eliminate the extra FAQs subsection here or is this how we're preventing multiple entries from opening for some nav items? If the latter, would it make sense to rename this one to "Deployment and Ops FAQs" and put "Licensing FAQs" as it's own subsection entry?

Reply:
FAQs is what @taroface suggested in the IA proposal after feedback from the field. I opened a [Jira issue](https://cockroachlabs.atlassian.net/browse/DOC-8280) to revisit the FAQs section and will work with Ryan to figure out what makes sense.

Reply:
In the IA proposal, this was just intended to be a top-level FAQs with the individual FAQs nested beneath:

<img width="277" alt="image" src="https://github.com/cockroachdb/docs/assets/8740013/d6533d0a-c8ce-45ee-a89d-95c4ad340b63">

I think this is just a glitch in the JSON nesting, unless there is some nuance I'm missing. Otherwise, I think we should fix the JSON here rather than in a separate issue.

Reply:
Edited.